Pipfile

```diff
@@ -4,48 +4,53 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-arxiv-auth = "==0.2.7"
```

Comment:
Interesting that arxiv-auth is not needed.

Reply:
Yes, it seems like it was future feature planning for the api or something.